src/components/TodoComponents/Todo.js

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+
+function Todo(props) {
+  return (
+    <div className="task">
+      <p className="list-number">{props.listNumber}.&nbsp;</p>
```

Comment:
nice touch having the number be listed as well

Reply:
Thanks!